miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/api/ApiClientSpec.kt

```diff
@@ -136,8 +136,7 @@ open class ApiClientSpec {
         ApiClient(
             baseUrl = TEST_URL_HTTPS_2,
             rasAppId = TEST_HA_ID_APP,
-            subscriptionKey = TEST_HA_SUBSCRIPTION_KEY,
-            hostAppVersionId = TEST_HA_ID_VERSION
+            subscriptionKey = TEST_HA_SUBSCRIPTION_KEY
         )
     }
 
```

Comment:
Also, we can remove-

```
private fun createApiClient(
        ...
        hostAppVersionId: String = TEST_HA_ID_VERSION,
        ...
    )
```

and, finally remove `TEST_HA_ID_VERSION` from `TestData`?